src/ClientActiveRequests.cc

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 1996-2020 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+#include "ClientActiveRequests.h"
+#include "client_side_request.h"
+#include "fde.h"
+#include "http/Stream.h"
+#include "ip/forward.h"
```

Comment:
It is unusual for a `.cc` file to include a `forward.h` file because the latter is usually needed for forward declarations. There are exceptions, of course, but please double check that this new `.cc` file needs this specific `#include`. If some other header file cannot compile in this context because that other header file is missing `ip/forward.h`, then this `.cc` file is not the place to fix that bug.